README.md

```diff
@@ -60,15 +60,15 @@ algorithms for signing your credentials. For now:
 
 Where to store the private keys?
 
-TODO: Add a brief discussion of where to store the private keys. Point to
-several recommended Wallet or KMS libraries.
+It is recommended to use a [Key Management Service](https://en.wikipedia.org/wiki/Key_management) such as:
+
+- [bedrock kms](https://github.com/digitalbazaar/bedrock-kms)
+- [amazon kms](https://aws.amazon.com/kms/)
 
-Use `await keyPair.export()`
 
 #### Publishing the Public Key
 
-TODO: Explain `documentLoader` / key resolvers, and where to put the public
-key so that the verifier can get to it.
+Public Keys can be stored in your [Key Management Service](https://en.wikipedia.org/wiki/Key_management), as static data on api end points, or even recreated from seeds. Regardless of which method that you use, your public key must be deferenced by a keyResolver so the verifier can use it to verify your credential. Your credential itself must be referenced by documentLoader.
```

Comment:
```suggestion
Public Keys can be stored in your [Key Management Service](https://en.wikipedia.org/wiki/Key_management), as static data on API end points, or even recreated from seeds. Regardless of which method that you use, your public key must be deferenced by a keyResolver so the verifier can use it to verify your credential. Your credential itself must be referenced by documentLoader.
```

Reply:
I can't parse that last bit.  Should that be "Your credential itself must be dereferenced by a documentLoader."?